webpack/scenes/ContentViews/components/EnvironmentLabels.js

```diff
@@ -11,15 +11,15 @@ const EnvironmentLabels = (environments) => {
       <React.Fragment key={env.id} style={{ marginBottom: '5px' }}>
         <Label
           color={labelColor}
-          isTruncated
```

Comment:
Did `isTruncated` get deprecated too?

Reply:
"Removed the isTruncated property from Label. This is now the default behavior. In addition, you can limit the text width using the new textMaxWidth property."
https://v5-archive.patternfly.org/get-started/upgrade/release-notes